src/stripQueryString.js

```diff
@@ -0,0 +1,3 @@
+export default function stripQueryString(url) {
+  return url.replace(/\?.*$/, '');
```

Comment:
you may also want to replace anchors, just in case:
```suggestion
  return url.replace(/[\?#].*$/, '');
```

Reply:
Good idea! I'll add this